src/components/Buttons/Button.js

```diff
@@ -238,10 +238,14 @@ const Button = props => {
       props.onMouseDown(e, props)
     },
     ...props.elementAttributes,
+    ...props.customElementProps,
```

Comment:
do we need this as well as `elementAttributes`?

Reply:
I think intentionally separating `attributes` and `props` is valuable (from a taxonomical perspective.) What do you think? :)